src/sql/workbench/browser/modelComponents/dropdown.component.ts

```diff
@@ -145,6 +145,10 @@ export default class DropDownComponent extends ComponentBase<azdata.DropDownProp
 			}
 			this._editableDropdown.enabled = this.enabled;
 			this._editableDropdown.fireOnTextChange = this.fireOnTextChange;
+
+			if (this.showInputTooltip) {
```

Comment:
Please submit this is a separate PR. Changes to core components (especially ones as widely used as dropdowns) should be reviewed separately to ensure they're properly reviewed.

Reply:
I've created a separate PR for the dropdown component. Once the other PR is merged, I will update this one.
https://github.com/microsoft/azuredatastudio/pull/16468

Reply:
removed the dropdown changes from this PR